plugins/inputs/intel_dlb/README.md

```diff
@@ -0,0 +1,118 @@
+driven programming model. In a polling model, lcores poll ethdev ports and
+associated Rx queues directly to look for a packet. By contrast in an event
+driven model, lcores call the scheduler that selects packets for them based on
+programmer-specified criteria. The Eventdev library adds support for an event
+driven programming model, which offers applications automatic multicore scaling,
+dynamic load balancing, pipelining, packet ingress order maintenance and
+synchronization services to simplify application packet processing.
+By introducing an event driven programming model, DPDK can support
+both polling and event driven programming models for packet processing,
+and applications are free to choose whatever model (or combination of the two)
+best suits their needs.
+
+## Pre-configuration
+
+Pre-configuration is required for plugin to be fully functional:
+
+- [DLB >= v7.4](https://www.intel.com/content/www/us/en/download/686372/intel-dynamic-load-balancer.html)
+- [DPDK >= 20.11.30](http://core.dpdk.org/download/)
+
+## Configuration
+
+```toml @sample.conf
+## Reads metrics from DPDK using v2 telemetry interface.
+[[inputs.intel_dlb]]
+  ## Path to DPDK telemetry socket.
+  # socket_path = "/var/run/dpdk/rte/dpdk_telemetry.v2"
+
+  ## Default eventdev command list, it gathers metrics from socket by given commands.
+  # eventdev_commands = ["/eventdev/dev_xstats", "/eventdev/port_xstats", "/eventdev/queue_xstats", "/eventdev/queue_links"]
+
+  ## Detect DLB devices based on device id.
+  # dlb_device_types = ["0x2710"]
+```
+
+Default configuration allows getting metrics for all metrics
+reported via `/eventdev/` command:
+
+- `/eventdev/dev_xstats`
+- `/eventdev/port_xstats`
+- `/eventdev/queue_xstats`
+- `/eventdev/queue_links`
+
+### Example configuration: default configuration
+
+This configuration will gather all metrics from given commands.
+This is default configuration of plugin.
+
+```toml
+[[inputs.intel_dlb]]
+  socket_path = "/var/run/dpdk/rte/dpdk_telemetry.v2"
+  eventdev_commands = ["/eventdev/dev_xstats", "/eventdev/port_xstats", "/eventdev/queue_xstats", "/eventdev/queue_links"]
+  dlb_device_types = ["0x2710"]
+```
+
+### Example configuration: specifying commands
+
+To gather metrics from specific commands we have to specify them in
+configuration file.This configuration will gather metrics from
+`"/eventdev/dev_xstats"` and `"/eventdev/queue_xstats"`.
+
+```toml
+[[inputs.intel_dlb]]
+  socket_path = "/var/run/dpdk/rte/dpdk_telemetry.v2"
+  eventdev_commands = ["/eventdev/dev_xstats", "/eventdev/queue_xstats"]
+  dlb_device_types = ["0x2710"]
+```
+
+### Example configuration: specifying socket path
+
+Path to telemetry socket may vary. In case you have to specify it,
+just change `socket_path` variable:
+
+```toml
+[[inputs.intel_dlb]]
+  socket_path = "/my-new/socket/path/dpdk_telemetry.v2"
+  eventdev_commands = ["/eventdev/dev_xstats", "/eventdev/queue_xstats"]
+  dlb_device_types = ["0x2710"]
+```
```

Comment:
This section is also not needed, this is described in the sample config

Reply:
Removed.